src/main/java/io/stargate/sgv2/jsonapi/service/operation/filters/table/codecs/CollectionCodecs.java

```diff
@@ -121,14 +174,29 @@ private static JSONCodec<Object, Object> findElementCodec(
     String msg =
         String.format(
             "no codec matching (list/set) declared element type `%s`, actual value type `%s`",
-            elementType, element.getClass());
+            elementType, element.getClass().getName());
+    throw new ToCQLCodecException(element, elementType, msg);
+  }
+
+  private static JSONCodec<Object, Object> findMapValueCodec(
```

Comment:
checking ToCQLCodecException is the correct exception, do we still hae MissingCodecException ?

Reply:
@amorton We do have `MissingCodecException`, but it seems to require `TableMetadata` and `ColumnMetadata` which is no longer available at this point (would need to be passed along). If necessary, could change.